libkbfs/data_types.go

```diff
@@ -160,29 +160,30 @@ const (
 // (duplicated) block.
 type BlockRefNonce [8]byte
 
-// zeroBlockRefNonce is a special BlockRefNonce used for the initial
+// ZeroBlockRefNonce is a special BlockRefNonce used for the initial
 // reference to a block.
-var zeroBlockRefNonce = BlockRefNonce([8]byte{0, 0, 0, 0, 0, 0, 0, 0})
+var ZeroBlockRefNonce = BlockRefNonce([8]byte{0, 0, 0, 0, 0, 0, 0, 0})
```

Comment:
It doesn't need exporting, although it seems strange to export a type and not export its zero value.

OTOH, this isn't a constant, which is annoying. Sigh.